### ISSUE TITLE
Change alt text for hero image in citizen engagement html file

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/website#3088]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will change the alt text for the hero image on [citizen engagement page](https://www.hackforla.org/citizen-engagement).

### Details
Designers and PMs identified that some alt texts in the Citizen Engagement page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] For the file `pages/citizen-engagement.html`, change [line 25](https://github.com/hackforla/website/blob/b834424fd5cf8962a01772533da45d30481e06aa/pages/citizen-engagement.html#L25) for the alt text:

from
```
alt="Image of people together in community" />
```
to
```
alt="" />
```
- [x] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3089 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3089 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3089 
- [Citizen Engagement page](https://www.hackforla.org/citizen-engagement)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `pages/citizen-engagement.html`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/b834424fd5cf8962a01772533da45d30481e06aa/pages/citizen-engagement.html#L25

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by raswani2023** [comment]
Availability: I am available most afternoons.
ETA: I have created a PR, just waiting to get it reviewed
